services/web/server/src/simcore_service_webserver/exporter/request_handlers.py

```diff
@@ -0,0 +1,87 @@
+# let us all thank code climate for this module
```

Comment:
not true anymore?

Reply:
This is still valid, the project_handlers had surpassed 500 lines of code, had to move the implementation elsewhere.